integration.js

```diff
@@ -95,54 +99,58 @@ function doLookup(entities, options, cb) {
         });
       } else if (translations) {
         translations.forEach(({ translatedText, detectedSourceLanguage }) => {
-          Logger.debug({ detectedSourceLanguage }, "Translation Data");
-          const sourceLanguage = getSourceLanguage(detectedSourceLanguage);
-
-          const details = {
-            ...result.body,
-            data: {
-              ...result.body.data,
-              translations: [{
-                translatedText,
-                detectedSourceLanguage: sourceLanguage
-              }]
-            }
-          };
+          Logger.debug({ detectedSourceLanguage }, 'Translation Data');
+          const inputAndOutputLanguageIsSame = detectedSourceLanguage === options.outputLanguage.value;
```

Comment:
@penwoodjon I don't think we have two many people translating into Hebrew but do we need to normalize the `options.outputLanguage.value` for Hebrew.

Maybe the easiest fix here is to use `iw` as the language code in the config.js and config.json for Hebrew?

Reply:
Changed in the config files!